src/main.d

```diff
@@ -274,18 +274,29 @@ int run(string[] args)
 	return 0;
 }
 
+
 string[] expandArgs(string[] args)
 {
+	// isFile can throw if it's a broken symlink.
+	bool isFileSafe(T)(T a){
```

Comment:
Please put opening braces on the next line the way it's done in the rest of the project.